modules/twinkleprod.js

```diff
@@ -142,20 +165,20 @@ Twinkle.prod.callback.prodtypechanged = function(event) {
 							label: 'Notify page creator if possible',
 							value: 'notify',
 							name: 'notify',
-							tooltip: 'Creator of ' + namespace + ' has to be notified.',
+							tooltip: 'Creator of article has to be notified.',
 							checked: true,
 							disabled: true
 						}
 					]
-				}
-			);
+				} );
 			//temp warning, can be removed down the line once BLPPROD is more established. Amalthea, May 2010.
 			var boldtext = document.createElement('b');
 			boldtext.appendChild(document.createTextNode('Please note that only unsourced biographies of living persons are eligible for this tag, narrowly construed.'));
 			field.append({
 				type: 'div',
 				label: boldtext
 			});
+			// TO DO: check if article is in Category:Living people, warn if not
```

Comment:
Isn't this #458?  Could use a rebase, I suppose.

Reply:
I just removed the cmt. Isn't it enough? I am not sure how to do a rebase.

Reply:
Eh, don't worry about it — I thought it was a comment present previously that got readded by merging in an old version, but it seems not.

Rebasing is either the greatest or worst thing ever in git: https://git-scm.com/docs/git-rebase